docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+services:
+  mongo:
+    build:
+      context: .
+      dockerfile: ./docker/dockerfile.mongodb-replica.yml
+    container_name: plasmidi-mongo
+    ports:
+      - '27018:27018'
+    networks:
+      - plasimidi-network
+    volumes:
+      # seed data can be created at ./init-mongo.js
+      # - ./init-mongo.js:/docker-entrypoint-initdb.d/init-mongo.js
+      # volume to persist data locally
+      - mongo-data:/data/db
+    restart: always
+    logging:
+      options:
+        max-size: 1g
+    environment:
+      MONGO_INITDB_ROOT_USERNAME: mongoadmin
+      MONGO_INITDB_ROOT_PASSWORD: secret
+      MONGO_REPLICA_PORT: 27018
+  server:
+    container_name: plasmidi-server
+    depends_on:
+      - mongo
+    build:
+      context: .
+      dockerfile: ${DOCKER_ENVFILE_PATH:-docker/dockerfile.dev.yml}
+    environment:
+      MONGODB_URL: mongodb://mongoadmin:secret@mongo:27018/plasmidi?directConnection=true&authSource=admin
```

Comment:
this is where a lot of the magic happens. If you wanted to deploy in prod using something like this you could use an env like line 30 above and replace this with an env variable or something and add a different command and boom.